.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: Run tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  run_tests:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install mamba & create environment
```

Comment:
Can you also add the pre-commit checks, please?

Reply:
Yep will do 👌🏼